src/sentry/conf/server.py

```diff
@@ -137,6 +137,7 @@ def env(
 SENTRY_STATISTICAL_DETECTORS_REDIS_CLUSTER = "default"
 SENTRY_METRIC_META_REDIS_CLUSTER = "default"
 SENTRY_ESCALATION_THRESHOLDS_REDIS_CLUSTER = "default"
+SENTRY_SPAN_BUFFER_CLUSTER = "default"
```

Comment:
There's a big warning [here](https://github.com/getsentry/sentry/pull/64435/files#diff-866fbc2da71a688b0a929236b2c864dfa8935d4a6423321d2d97a28337c6e6e8R124-R126). Does this need to be addressed even if none of this runs in prod yet?

Reply:
Not until the consumers whatever connects to Redis is in prod.